spec/spec_helper.rb

```diff
@@ -11,35 +12,29 @@
 require 'napa'
 require 'napa/rspec_extensions/response_helpers'
 
+module NapaSpecClassHelpers
+  # Apparently Thor's last name is Odinson... who knew?!
+  # http://marvel.com/universe/Thor_(Thor_Odinson)
+  # http://www.infinitelooper.com/?v=4D7cPH7DHgA#/170;230
+  def tell_me_mr_odinson_what_good_is_a_phone_call_if_youre_unable_to_speak?
```

Comment:
Fair enough, I probably predisposed my self to interpret it that way, due to my own emotional immaturity -- primarily a distrust/dislike of authority, which shades my view of anyone who seems to express it. That's my shit, not your fault, no need to apologize. I'll put more emphasis in my brain on remembering everyone's humanity and giving them the benefit of the doubt :)

But yeah, I like this method name. It gives me a sense of serenity. I have no problem with you changing it to better suit your coding standards, but if I have to be the one to do it, it'll feel like an imposition. So, seems best for both of us.